src/Microsoft.DotNet.XHarness.CLI/Commands/WASM/Browser/WasmTestBrowserCommand.cs

```diff
@@ -108,12 +106,21 @@ protected override async Task<ExitCode> InvokeInternal(ILogger logger)
                 "Cannot start the driver service"
             };
 
+            foreach (var file in Directory.EnumerateFiles(_arguments.OutputDirectory, "chromedriver-*.log"))
+                File.Delete(file);
```

Comment:
this is probably not really necessary since on helix each work item gets a new output directory anyway and locally you don't care if you have multiple files.

Reply:
Just useful to to not have stale files, when you are trying to do multiple locals runs, and check if there were any logs produced, and/or if they had anything useful.